src/pages/Registration/Registration.scss

```diff
@@ -0,0 +1,9 @@
+.registration_container {
+    margin: auto;
+    margin-top: 2vh;
+
+    h1 {
+        display: flex;
```

Comment:
learn about mixins, so this can be reduced

Reply:
ok